.travis.yml

```diff
@@ -6,10 +6,6 @@ env:
 matrix:
   fast_finish: true
   include:
-    - env: DM=~6.0
-      php: 5.3
-    - env: DM=~6.0
```

Comment:
You are removing all tests with DataModel 6 here. You need to move these version numbers down, at least one level, so every version is at least tested once.